sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/IdentityDetails.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> The IdentityDetails. </summary>
+    public partial class IdentityDetails
```

Comment:
IdentityDetails is too general, please rename it to a more specific one.

Reply:
@live1206 can you pls share an example if IdentityDetails has been renamed in another Azure service ? This feature is to support User Assigned Managed Identity for backup vaults

Reply:
As offline communicated, this could potentially conflict with other services. Even they locate in different namespaces, it could confuse user which namespace/package to install for the type with same name.